todo/addItem.php

```diff
@@ -2,7 +2,9 @@
     include '../inc/db.inc.php';
     if(isset($_GET['t'])){
       $todo = $_GET['t'];
-      $sql = "INSERT INTO todo(text) VALUES($todo)";
+      $familyID = $_GET['family'];
+      $userID = $_GET['user'];
+      $sql = "INSERT INTO todo(title, time, user_id, family_id) VALUES($todo, null, $userID, $familyID)";
 
       if ($con->query($sql) === FALSE) {
```

Comment:
denne trenger du ikke å gjenta to ganger; hvis du har den en gang til slutt utenfor blockene så går det fin:) fordi variablene er globale you know:)))))